ScrollViewAssignment/ScrollViewAssignment/CustomTableViewCell.swift

```diff
@@ -7,44 +7,36 @@
 
 import UIKit
 
-protocol CustomTableViewCellDelegate: AnyObject {
-    func minusAction (count: Int, cell: CustomTableViewCell)
-    func plusAction (count: Int, cell: CustomTableViewCell)
-}
+//protocol CustomTableViewCellDelegate: AnyObject {
+//    func minusAction (cell: CustomTableViewCell)
+//    func plusAction (cell: CustomTableViewCell)
+//}
 
 class CustomTableViewCell: UITableViewCell {
     
-    weak var delegate: CustomTableViewCellDelegate?
+//    weak var delegate: CustomTableViewCellDelegate?
     
     @IBOutlet weak var imageOfCell: UIImageView!
     @IBOutlet weak var headerLabelOfCell: UILabel!
     @IBOutlet weak var lowerLabelOfCell: UILabel!
     @IBOutlet weak var countLabel: UILabel!
+    @IBOutlet weak var plusButton: UIButton!
+    @IBOutlet weak var minusButton: UIButton!
     
-    @IBAction func minusButton(_ sender: Any) {
-        if count > 0 {
-            count -= 1
-
-        }
-        delegate?.minusAction(count: count, cell: self)
-    }
-    @IBAction func plusButton(_ sender: Any) {
-        count += 1
-        delegate?.plusAction(count: count, cell: self)
-
+    func itemForCell(item: ShopItem) {
+        imageOfCell.image = UIImage(named: item.nameOfimage)
+        headerLabelOfCell.text = item.name
+        lowerLabelOfCell.text = item.description
     }
     
-    var count = 0
-
-    override func awakeFromNib() {
-        super.awakeFromNib()
-        // Initialization code
+    var complation: ((_ : Any) -> ())?
```

Comment:
you don't need to pass button, just pass smallest piece of information you need, for example: isPlusAction: Bool